policy-supplemental/platforms.md

```diff
@@ -60,6 +60,13 @@
 | linux-ppc64le           |        | Linux              |        | ppc64 little endian     |        | gcc             |        | \@dannytsen \@erichte-ibm \@naynajain                   |
 | linux64-riscv64         |        | Linux              |        | riscv64                 |        | gcc             |        | \@ZenithalHourlyRate                                    |
 | linux32-riscv32         |        | Linux              |        | riscv32                 |        | gcc             |        | \@ZenithalHourlyRate                                    |
+| aix-gcc                 |        | AIX                |        | ppc32                   |        | gcc             |        | \@sanumesh                                              |
+| aix64-gcc               |        | AIX                |        | ppc64                   |        | gcc             |        | \@sanumesh                                              |
+| aix64-gcc-as            |        | AIX                |        | ppc64                   |        | gcc as directive|        | \@sanumesh                                              |
+| aix-cc                  |        | AIX                |        | ppc32                   |        | IBM xlc         |        | \@sanumesh                                              |
+| aix64-cc                |        | AIX                |        | ppc64                   |        | IBM xlc         |        | \@sanumesh                                              |
+| aix-clang               |        | AIX                |        | ppc32                   |        | IBM openxl clang|        | \@sanumesh                                              |
+| aix64-clang             |        | AIX                |        | ppc64                   |        | IBM openxl clang|        | \@sanumesh                                              |
```

Comment:
There are Configure targets, aix-cc-solib, and aix64-cc-solib that appear in the Configuration output, that appear to have never made it into this document.  Should those be added to this list as well, or should they be removed as Configure targets?

Reply:
They should be added here either in the community platforms or into the unadopted. Platforms should be in general removed only on major level boundary (exceptions are possible if a platform is clearly broken and/or useless).

Reply:
aix-cc-solib, and aix64-cc-solib  seems to be more customer specific requirement on the way libraries are packaged after the build.So, I can keep it as unadapted platform

Reply:
Can you please review ? Thanks